src/unseal.js

```diff
@@ -0,0 +1,24 @@
+const axios = require('axios');
+
+const { VAULT_KEYS, VAULT_ADDR } = process.env;
```

Comment:
Might it be worth using the `convict` library to handle environment variables? You'll get better errors for missing configuration that way and also the ability to set default for VAULT_ADDR.

Reply:
Switched to that